app/components/works/agreement_component.html.erb

```diff
@@ -2,11 +2,16 @@
   <header>Terms of Deposit *</header>
   <div class="row">
     <div class="form-check col-auto">
-      <%= form.check_box :agree_to_terms, required: true, class: 'form-check-input' %>
-      <%= form.label :agree_to_terms,
-       "I agree to the #{link_to 'SDR Terms of Deposit', '#termsOfDepositModal', data: { bs_toggle: 'modal', bs_target: '#termsOfDepositModal'}}".html_safe,
-       class: 'form-check-label' %>
-       <div class="invalid-feedback">You must agree to the Terms of Deposit.</div>
+      <% if form.object.agree_to_terms %>
```

Comment:
Can you make this into a method on the `AgreementComponent` so that it's just `agreed_to_terms?` here?

Reply:
add a helper method on the agreement component to make this more readable